DXRFixes/DeusEx/Classes/ScriptedPawn.uc

```diff
@@ -663,6 +692,33 @@ state Sitting
 
 }
 
+// returns true if the damage type can knock the pawn out
+function bool CanKnockUnconscious(coerce string damageType)
+{
+    if (Robot(self) != None) {
+        return false;
+    }
+
+    // small creatures can't survive being hit by JC
+    if (mass <= 10.0) {
+        if (
+            (damageType == "Stunned") ||
+            (damageType == "Poison") ||
+            (damageType == "PoisonEffect")
+        ) {
```

Comment:
`KnockedOut` isn't here for animals?

Reply:
KnockedOut is the damage type done by the baton, and only the baton. The purpose here is to not let small creatures survive a hit from a baton.